### PR TITLE
update prerelase and add release

### PR DESCRIPTION
**Pull request by arvdias**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        with:
+          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+      - name: Publish to S3 action
+        uses: newrelic/infrastructure-publish-action@main
+        env:
+          WORKDIR: /home/gha
+          AWS_S3_BUCKET_NAME: "nr-downloads-main"
+          AWS_S3_LOCK_BUCKET_NAME: "onhost-ci-lock"
+          AWS_REGION: "us-east-1"
```
Comment (gsanchezgavier) [inline .github/workflows/release.yml:33]: why are these passed as env ?

**Review by gsanchezgavier — APPROVED** [review]
lgtm

File: .github/workflows/release.yml (hunk 1 above)
Comment (arvdias) [reply .github/workflows/release.yml:33]: just to not have them directly in the configuration of the action to be consistent with other cases. We should decide if we want to use env vars for everything or put values directly in the action config

Comment (arvdias) [reply .github/workflows/release.yml:32]: I've changed to use env vars for everything

Comment (roobre) [reply .github/workflows/release.yml:32]: I'm not amused with the mix, but at this point I just want to let this see the light so I'm dropping my perfectionism for once :(

**Review by roobre — APPROVED** [review]

**Comment by roobre** [comment]
Just one final comment: Do we have documentation on how to deploy this action in other repos? Which vars should change, what are their meaning, etc. If we do, might be worth pointint to it with a comment just on top of the `uses` block for future reference.

File: .github/workflows/release.yml (hunk 1 above)
Comment (arvdias) [reply .github/workflows/release.yml:32]: now it's consistent! :) it's all in env vars :)

**Comment by arvdias** [comment]
> Just one final comment: Do we have documentation on how to deploy this action in other repos? Which vars should change, what are their meaning, etc. If we do, might be worth pointint to it with a comment just on top of the `uses` block for future reference.

No we don't. It's 99.9% copy-paste. You just have to be sure to change the name of thee integration in the "release.yml" workflow (ex: haproxy -> postgresql)

**Pull request merged**